JS/test/airport.spec.js

```diff
@@ -1,104 +1,103 @@
-            if (militaryPlane.getMilitaryType() === MilitaryType.TYPE_TRANSPORT) {
-                flag = true;
-                break;
-            }
-        }
-        assert.equal(flag,true);
-    });
-
-    it('should check passenger plane with max capacity', () => {
-        let airport = new Airport(planes);
-        let expectedPlaneWithMaxPassengersCapacity = airport.getPassengerPlaneWithMaxPassengersCapacity();
-        assert.isFalse( expectedPlaneWithMaxPassengersCapacity == planeWithMaxPassengerCapacity);
-    });
-
-
-    it('test 3', () => {
-        console.log("TEST testGetPassengerPlaneWithMaxCapacity started!");
-        let airport = new Airport(planes);
-        airport.sortByMaxLoadCapacity();
-        let planesSortedByMaxLoadCapacity = airport.getPlanes();
-        let nextPlaneMaxLoadCapacityIsHigherThanCurrent = true;
-        for (let i = 0; i < planesSortedByMaxLoadCapacity.length - 1; i++) {
-            let currentPlane = planesSortedByMaxLoadCapacity[i];
-            let nextPlane = planesSortedByMaxLoadCapacity[i + 1];
-            if (currentPlane.getMinLoadCapacity() > nextPlane.getMinLoadCapacity()) {
-                nextPlaneMaxLoadCapacityIsHigherThanCurrent = false;
-                break;
-            }
-        }
-        assert.isTrue(nextPlaneMaxLoadCapacityIsHigherThanCurrent);
-    })
-
-    it('testHasAtLeastOneBomberInMilitaryPlanes', () => {
-        let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getBomberMilitaryPlanes ();
-        let flag = false;
-        for (let militaryPlane of bomberMilitaryPlanes) {
-            if (militaryPlane.getMilitaryType() === MilitaryType.BOMBER) {
-                flag = true;
-            }
-            else{
-                assert.fail("Test failed!");
-            }
-        }
-
-        // if not failed;
-    })
-
-    it('should check that experimentsl planes has classification level higher than unclassified', () => {
-        let airport = new Airport(planes);
-        let bomberMilitaryPlanes  = airport.getExperimentalPlanes ();
-        let hasUnclassifiedPlanes  = false;
-        for (let experimentalPlane  of bomberMilitaryPlanes) {
-            if (experimentalPlane.classificationLevel === ClassificationLevel.UNCLASSIFIED) {
-                hasUnclassifiedPlanes = true;
-
-        }
-        assert.isFalse(hasUnclassifiedPlanes);
-
-        }
-    });
-
+const assert = require("chai").assert;
+
+const Plane = require("../Planes/Plane");
+const MilitaryPlane = require("../Planes/MilitaryPlane");
+const PassengerPlane = require("../Planes/PassengerPlane");
+const Airport = require("../Airport");
+const MilitaryType = require("../Models/MilitaryType");
+const ExperimentalPlane = require("../Planes/ExperimentalPlane");
+const ExperimentalTypes = require("../Models/ExperimentalTypes");
+const ClassificationLevel = require("../Models/ClassificationLevel");
+
+describe("My Test", () => {
```

Comment:
What does this test description give to the reader? It's uninformative. Please correct.